storage/gc_queue_test.go

```diff
@@ -91,22 +91,22 @@ func TestGCQueueShouldQueue(t *testing.T) {
 		// No GC'able bytes, with intent age=1/2 period, and other 1/2 period elapsed.
 		{0, 0, 1, ia / 2, now, false, 0},
 		// No GC'able bytes, with (abs and avg) intent age=1.5*normalization.
-		{0, 0, 1, 3 * ia / 2, now, true, 1.5},
+		{0, 0, 1, 3 * ia / 2, now, true, 1.5 * considerThreshold},
 		// No GC'able bytes, 2 intents, with avg intent age=3.5*normalization.
-		{0, 0, 2, 7 * ia, now, true, 3.5},
+		{0, 0, 2, 7 * ia, now, true, 3.5 * considerThreshold},
 		// GC'able bytes, no time elapsed.
 		{bc, 0, 0, 0, now, false, 0},
 		// GC'able bytes, avg age = just below TTLSeconds.
 		{bc, bc*ttl - 1, 0, 0, now, false, 0},
 		// GC'able bytes, avg age = 2*TTLSeconds.
-		{bc, 2 * bc * ttl, 0, 0, now, true, 2},
+		{bc, 2 * bc * ttl, 0, 0, now, true, 2 * considerThreshold},
 		// x2 GC'able bytes, avg age = TTLSeconds.
-		{2 * bc, 2 * bc * ttl, 0, 0, now, true, 2},
+		{2 * bc, 2 * bc * ttl, 0, 0, now, true, 2 * considerThreshold},
 		// GC'able bytes, intent bytes, and intent normalization * 2 elapsed.
 		// Queues solely because of gc'able bytes.
-		{bc, 5 * bc * ttl, 10 * ia, 0, now, true, 5},
+		{bc, 5 * bc * ttl, 10 * ia, 0, now, true, 5 * considerThreshold},
 		// A contribution of 1 from gc, 10/5 from intents.
-		{bc, bc * ttl, 5, 10 * ia, now, true, 1 + 2},
+		{bc, bc * ttl, 5, 10 * ia, now, true, (1 + 2) * considerThreshold},
```

Comment:
You could move these multiplications down to the point where you compare against `test.priority` so it's not repeated on every line.